fbw-a32nx/mach.config.js

```diff
@@ -3,7 +3,16 @@ const postCssPlugin = require('esbuild-style-plugin');
 const tailwind = require('tailwindcss');
 const postCssColorFunctionalNotation = require('postcss-color-functional-notation');
 const postCssInset = require('postcss-inset');
-const { typecheckingPlugin } = require("#build-utils");
+const { typecheckingPlugin, generateInstrumentsMetadata, getMachInstrumentBuilders } = require("#build-utils");
+const path = require("path");
+
+const {
+    msfsAvionicsInstrument,
+    reactInstrument,
+} = getMachInstrumentBuilders({
+    templateIDPrefix: 'A32NX',
```

Comment:
```suggestion
    templateIDPrefix: 'A32NX',
```

I'd actually like to read from .env, or some kind of markup here, but I'm not sure if its actually possible.